OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -142,6 +142,8 @@ private void HandleBasicWarp(PlayerSpawner spawner, SpawnPoint[] spawnPoints)
             oxygenController.UpdateOxygen();
             var ship = GameObject.FindGameObjectWithTag("Ship");
             ship.SetActive(false);
+            var villageMusicController = FindObjectOfType<VillageMusicVolume>();
+            villageMusicController.OnEffectVolumeExit(spawner.gameObject);
```

Comment:
Have you tested spawning elsewhere and then going to the village during the day? This API's a bit confusing, but it seems like it might fully disable the village music, even if you go back.

Reply:
Yes I tested that case. The reason this code works is that it calls the method for the player leaving the village.

Reply:
Great! LGTM, although we'll probably want to make a list of spawn points for which we don't call this method, like the regular starting campfire.

Reply:
yes I agree, I did not add that since we're going to expand on the PR david has open right now in a bit, so we have a dict of spawn point to pretty names. We can add that field to that. Though if we're going to have more than just pretty names, it might be worth making a struct for it and having a list of them or something since the fast lookup is likely irrelevant for 50 items.

Reply:
It looks like this might break the music if we *do* decide to spawn the player on Timber Hearth. Are we ok with that?